projects/fsastorefrontlib/src/cms-components/my-account/order/order-details/order-detail-items/order-detail-items.component.html

```diff
@@ -1,7 +1,15 @@
 <ng-container *ngIf="order$ | async as order">
-  <h2 class="heading-headline py-3 mt-3 mb-5 text-center">
-    {{ 'orderDetails.orderId' | cxTranslate }} {{ order?.code }}
-  </h2>
+  <div class="d-flex align-items-center">
+    <a class="h5 py-3 mt-3 mb-5" routerLink="/my-account/orders/"
+      ><span class="d-none d-md-block">{{ 'common.back' | cxTranslate }}</span>
+      <span class="h6 d-md-none fas fa-arrow-left"></span
+    ></a>
+    <h2 class="heading-headline py-3 mt-3 mb-5 text-center col-11">
```

Comment:
If you need col-11 here then it must be wrapped in `row` class and the sibling element has to have col-1 then